lib/rubocop/target_finder.rb

```diff
@@ -96,7 +96,10 @@ def find_files(base_dir, flags)
                 else
                   # Search the non-excluded top directories, but also add files
                   # on the top level, which would otherwise not be found.
-                  "{#{base_dir}/*,{#{wanted_toplevel_dirs.join(',')}}/**/*}"
+                  escaped_toplevel_dirs = wanted_toplevel_dirs.map do |dir|
+                    dir.gsub(',', '\\,')
```

Comment:
Oops. I need to verify that I got the single/double escaping within single/double quotes thing right.